certified-connectors/secplugs/readme.md

```diff
@@ -1,20 +1,20 @@
 
 ### NOTE
-> This is a Secplugs scan connector. The connector is provided here with the intent to illustrate certain features and functionality around connectors.
+> This is a Scancloud APIs connector. The connector is provided here with the intent to illustrate certain features and functionality around connectors.
 
-## Secplugs Scan Connector
-Secplugs provides a powerful platform for pluggable security using secplugs plugins.
+## Scancloud APIs Connector
+Scancloud APIs provides a powerful platform for pluggable security using Scancloud APIs plugins.
 
 
 
 ## Prerequisites
 You will need the following to proceed:
 * A Microsoft Power Apps or Power Automate plan with custom connector feature
-* An account in secplugs
+* An account in Scancloud
 * The Power platform CLI tools
 
 ## Building the connector 
-You would need to ensure that you have an account in secplugs and using https://secplugs.com/index.php/providers an power automate profile is created.
+You would need to ensure that you have an account in secplugs and using https://scancloud.io/index.php/providers an power automate profile is created.
 After that is completed, you can create and test the secplugs connector.
 
 ### Deploying the sample
```

Comment:
@Ajaatsecplugs please format the readme document according to [the readme.md template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md).

Reply:
Fixed